__testHelpers__/test.helper.js

```diff
@@ -0,0 +1,27 @@
+// You may obtain a copy of the License at
```

Comment:
Please update this header to use the standard BC Gov header the project is using; you can find it in any one of the existing `.js` files. Jus make sure the date is updated to `.2019`